front/src/app/welcome/map/map.component.html

```diff
@@ -29,7 +29,7 @@
           </mat-option>
         </mat-optgroup>
       </mat-autocomplete>
-      <mat-hint>Une adresse, une parcelle, un point d'intérêt, etc.</mat-hint>
+      <mat-hint>Une adresse, une parcelle, une commune, etc..</mat-hint>
```

Comment:
```suggestion
      <mat-hint>Une adresse, une parcelle, une commune, etc...</mat-hint>
```

Doesn't it take 3 points?

Reply:
It takes one :)